speech_to_text/speech_to_text.livemd

```diff
@@ -0,0 +1,217 @@
+# Speech to text
+
+```elixir
+Logger.configure(level: :info)
+
+Mix.install(
+  [
+    :bumblebee,
+    :exla,
+    :membrane_core,
+    :membrane_portaudio_plugin
+  ],
+  config: [
+    nx: [default_backend: EXLA.Backend]
+  ]
+)
+```
+
+## Prerequisites
+
+On Linux, you need to install PortAudio, for example with `sudo apt install portaudio19-dev`.
```

Comment:
I would point to the installation section in the `membrane_portaudio_plugin`'s README (the manual there covers multiple OS and "is aware" of precompiled deps)